### PR TITLE
os-sunnyvalley: description update

### PR DESCRIPTION
**Pull request by muratbalaban43**
Make the plugin description more informative

### CODE REVIEW

File: vendor/sunnyvalley/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PLUGIN_NAME=		sunnyvalley
-PLUGIN_VERSION=		1.1
-PLUGIN_COMMENT=		Sunny Valley Networks Vendor Repository
+PLUGIN_VERSION=		1.2
+PLUGIN_COMMENT=		Vendor repository for Sensei (Next Generation Firewall Extensions for OPNsense)
```
Comment (fichtner) [inline vendor/sunnyvalley/Makefile]:
```suggestion
PLUGIN_COMMENT=		Vendor repository for Sensei (Next Generation Firewall Extensions)
```

File: vendor/sunnyvalley/pkg-descr [hunk 2]
```diff
@@ -1,13 +1,18 @@
-This plugin adds a proprietary repository to install
-Sensei, a plugin for OPNsense complementing the firewall
-with state of the art next generation firewall features
-like:
+This plugin adds a proprietary repository to install Sensei, a plugin
+for OPNsense, complementing the firewall with state of the art 
+next generation firewall features.
+
+You'll need to install os-sensei plugin after installing this repo plugin.
```
Comment (fichtner) [inline vendor/sunnyvalley/pkg-descr]:
```suggestion
You will need to install os-sensei plugin after installing this repo plugin.
```

**Comment by muratbalaban43** [comment]
@fichtner , thanks :)

**Comment by fichtner** [comment]
Seeing the suggestions as resolved but not implemented... was that intended? :)

**Comment by muratbalaban43** [comment]
@fichtner , sorry, dinosaur svn user here :) Thought github would automagically merge the changes. New commit coming.

**Comment by muratbalaban43** [comment]
@fichtner , I guess it's ok this time :)

**Pull request merged**